mllm/models/smollm3_3B/tokenization_smollm3.hpp

```
@@ -0,0 +1,185 @@
+
+    return ret;
+  }
+
+  ARGenerationOutputPast convertMessage(const SmolLM3Message& message) {
+    // 选择模板
+    std::string template_str = message.enable_thinking 
+                               ? SmolLM3Message::thinking_template 
+                               : SmolLM3Message::direct_template;
+
+    // 替换prompt占位符
+    std::string applied_string = template_str;
```

Clang (14.0.6)

[error] variable 'applied_string' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)